database/deepdive-load

```diff
@@ -180,5 +180,8 @@ done
 # load everything queued so far
 loadQueued
 
+# always analyze relation after loading
+deepdive db analyze "$Relation"
```

Comment:
Might be better to do `deepdive-db analyze` given the `deepdive-mark done` pattern?